account_move_base_import/models/account_journal.py

```diff
@@ -61,6 +61,22 @@ class AccountJournal(models.Model):
         help="Tic that box to automatically launch the completion "
         "on each imported file using this journal.")
 
+    create_counterpart = fields.Boolean(
+        string="Create Counterpart",
+        help="Tic that box to automatically create the move counterpart")
+
+    split_counterpart = fields.Boolean(
+        string="Split Counterpart",
+        help="Two counterpart will be automatically created one for "
+             "the refunds and one for the payments")
+
+    @api.onchange('import_type')
```

Comment:
I made an onchange because I think this option depend exclusively of the parser, so it depends of the import type.
But anyway, I don't mind setting it by default instead.